parsons/google/google_bigquery.py

```diff
@@ -0,0 +1,89 @@
+from google.cloud.bigquery import Client
+from parsons import Table
+from parsons.google.utitities import setup_google_application_credentials
+from parsons.utilities.files import create_temp_file
+import petl
+import pickle
+
+
+class BigQuery:
```

Comment:
So, I didn't know this until pylint told me a couple weeks ago, but the `(object)` is unnecessary for all classes in Python 3. So `class BigQuery:` and `class BigQuery(object):` mean exactly the same thing.

In Python 2, there was this old-style/[new-style](https://docs.python.org/2/glossary.html#term-new-style-class) classes thing, and if you didn't specify `(object)` then the class would be old-style (which you pretty much never wanted). When they did Python 3, they got rid of the old-style classes, and so there was no need to specify the `(object)` since all classes inherited from `object`.

Reply:
I'm happy to add it in for consistency with the other code. Either way.

Reply:
It's fine as is for now.